scratch/Functional_Performance/do_it_functions.py

```diff
@@ -37,35 +47,23 @@ def calc_it_metrics_sites(inputs_df, source, sink, site, log_transform):
            'tmmx', 'velocity', 'vs']]
     targets_site = inputs_df.loc[site][['do_min','do_mean','do_max']]
     targets_site['do_range'] = targets_site['do_max']-targets_site['do_min']
-    
-    #0 baseline LSTM
-    v0_bl = pd.read_feather('scratch/4_func_perf/in/results_tmmx_tmmn/models/0_baseline_LSTM/nstates_10/nep_100/rep_0/preds.feather')
-    bl = v0_bl[v0_bl['site_id'] == site].set_index('date')[['do_min','do_mean','do_max']]
-    bl['do_range'] = bl['do_max']-bl['do_min']
-
-    #1 metab 
-    v1_multi = pd.read_feather('scratch/4_func_perf/in/results_tmmx_tmmn/models/1_metab_multitask/nstates_10/nep_100/rep_0/preds.feather')
-    multi = v1_multi[v1_multi['site_id'] == site].set_index('date')[['do_min','do_mean','do_max']]
-    multi['do_range'] = multi['do_max']-multi['do_min']
-
-    #2 multi-task
-    v1a_metab = pd.read_feather('scratch/4_func_perf/in/results_tmmx_tmmn/models/1a_multitask_do_gpp_er/nstates_10/nep_100/rep_0/preds.feather')
-    metab = v1a_metab[v1a_metab['site_id'] == site].set_index('date')[['do_min','do_mean','do_max']]
-    metab['do_range'] = metab['do_max']-metab['do_min']
-    
-    #3 multi-task dense
-    v2_metab_dense = pd.read_feather('scratch/4_func_perf/in/results_tmmx_tmmn/models/2_multitask_dense/nstates_10/nep_100/rep_0/preds.feather')
-    metab_dense = v2_metab_dense[v2_metab_dense['site_id'] == site].set_index('date')[['do_min','do_mean','do_max']]
-    metab_dense['do_range'] = metab_dense['do_max']-metab_dense['do_min']
 
-    #create targets dictionary
-    tar_dict = {'observed':targets_site, 'baseline':bl,
-                'multitask':multi, 'metab':metab, 'metab_dense':metab_dense}
+    tar_dict = {}
+    for m in models:
+        file_path = f'{base_file_path}/{m}/nstates_10/nep_100/rep_{replicate}/preds.feather'
```

Comment:
I wasn't sure how to best deal with the filepaths. I left some of this hard-coded, but maybe should have done it all dynamic. I left the `nstates_10/nep_100/rep_{replicate}/preds.feather` hard-coded.